packages/rs-platform-version/src/version/drive_abci_versions.rs

```diff
@@ -78,6 +78,15 @@ pub struct DriveAbciValidationVersions {
     pub state_transitions: DriveAbciStateTransitionValidationVersions,
     pub process_state_transition: FeatureVersion,
     pub state_transition_to_execution_event_for_check_tx: FeatureVersion,
+    pub penalties: PenaltyAmounts,
+}
+
+/// All of these penalty amounts are in credits
+#[derive(Clone, Debug, Default)]
```

Comment:
why not?

Reply:
Because this tree is going to be too big since it mixes two different entities: versions and constants. If we want to have constants based on version I would introduce a separate tree for that.

Reply:
We can make a different PR for that, but we can even do it after mainnet, because it wouldn't be a breaking change.